docs_src/options/callback/tutorial005.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Optional
+
+import typer
+
+
+def names_callback(ctx: typer.Context, param: typer.CallbackParam, value: str):
```

Comment:
```suggestion
def names_callback(ctx: typer.Context, param: typer.CallbackParam, value: str):
```
`value` should be of type `Optional[List[str]]` here